deploy/firesim

```diff
@@ -182,6 +182,8 @@ def managerinit(args: argparse.Namespace):
             "managerinit replace start",
             "managerinit replace end",
             bf_recipe_lines)
+    elif args.platform == 'vitis':
+        rootLogger.info(f"Skipping default config overrides for vitis due to experimental support.")
```

Comment:
Can you replace this with copying in the correct stuff?

Reply:
Done